azure/azure-explanation/confidential-computing.rst

```diff
@@ -29,14 +29,10 @@ In short, a confidential VM is a combination of two features:
 
 It's important to note that memory encryption is always enabled with a confidential VM, but FDE is optional and requires explicit activation after the VM is provisioned.
 
-
 Using Ubuntu on confidential VMs
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Confidential VMs require the use of special instance sizes and a special version of Ubuntu.
-
-* A list of instance sizes that can be used for confidential VMs is given in `Azure's documentation <https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview>`_ 
-* Only `this specific offer of Ubuntu <https://azuremarketplace.microsoft.com/en-gb/marketplace/apps/canonical.0001-com-ubuntu-confidential-vm-focal>`_ supports confidential VMs.
-
+Currently there are two Ubuntu images which support Confidential VMs:
 
-   
\ No newline at end of file
+* Ubuntu CVM 20.04 LTS, which supports SEV-SNP
+* Ubuntu CVM 22.04 LTS, which supports both SEV-SNP and Intel TDX (in public preview, yet to reach General Availability)
```

Comment:
> (in public preview, yet to reach General Availability)

Is this only for Intel TDX support or for 22.04 as a whole?

Reply:
Intel TDX. I was considering putting more context for Intel TDX as part of this commit, but given the GA for TDX has been pushed back 8-10 months this can be handled in greater detail later.